scalafix-tests/unit/src/test/scala/scalafix/tests/core/util/TokenListTest.scala

```diff
@@ -0,0 +1,30 @@
+package scalafix.tests.core.util
+
+import org.scalatest.FunSuite
+import scala.meta._
+import scalafix.util.TokenList
+
+class TokenListTest extends FunSuite {
+
+  test("Prev returns the firstToken when is given the firstToken") {
+    val tokens = "".tokenize.get // this contains two tokens: beginningOfFile and endOfFile
+    val firstToken = tokens.last
+    val tokenList = new TokenList(tokens)
+    assert(tokenList.prev(firstToken) == firstToken)
+    assert(tokenList.prev(tokens.last) == firstToken)
+  }
+
+  test("Slice gives an empty list with same token as inputs") {
+    val tokens = "val x = 2".tokenize.get
```

Comment:
do we need `val x = 2`? It seems like we could reuse the same `"".tokenize.get` by putting in the top of the suite before the first `test`. That would to write small and declarative tests without duplicating `new TokenList(..)` in every unit test.